Source/WebCore/page/ios/FrameIOS.mm

```diff
@@ -392,17 +392,17 @@
 
 static bool nodeIsMouseFocusable(Node& node)
 {
-    if (!is<Element>(node))
+    RefPtr element = dynamicDowncast<Element>(node);
```

Comment:
We should not need a RefPtr here. The caller should have ref'd the parameter for us and you merely casted it.